samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -249,6 +249,42 @@ Configure the Sensor Setup Server model on the **Mesh Sensor** node:
 
 The Sensor Setup Server model is now configured and able to receive sensor setting messages from the Sensor Client.
 
+Occupancy Mode
```

Comment:
While reading this, I realized that we have not explained where does this "standby" level comes from or what to expect when light goes to standby. So, I checked above and noticed some missing info as follows:


Line 228 above:

> " to 0% over five seconds" 

Should be: 
"to `CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_STANDBY`"

> The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG` is 10000 (~15%).

Should be:
"The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG` is 10000 (~15%).
The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_STANDBY` is 0 (0%)."